src/main/java/fi/aalto/cs/apluscourses/presentation/exercise/ExerciseViewModel.java

```diff
@@ -18,6 +18,38 @@ public String getPresentableName() {
     return APlusLocalizationUtil.getEnglishName(getModel().getName());
   }
 
+  /**
+   * Returns {@code true} if the exercise is submittable from the plugin, {@code false} otherwise.
+   */
+  public boolean isSubmittable() {
+    String presentableName = getPresentableName();
+    return presentableName.length() > "Assignment xx (".length()
```

Comment:
Do we have a plan for how this will be eventually unhacked?

Reply:
Nothing concrete yet at this point. Checking submittability requires making a request to the individual exercise API end point, so most likely some sort of caching would be highly preferable at that point.